OpenMEEG/libs/OpenMEEGMaths/include/matrix.h

```diff
@@ -187,7 +187,7 @@ namespace OpenMEEG {
     }
     
     inline double Matrix::frobenius_norm() const {
-    #ifdef HAVE_LAPACK
+    #if defined(HAVE_LAPACK) && !defined(NO_DLANGE)
```

Comment:
I just had difficulties finding the dlange in the mkl, but we only use it to get the frobenius norm in very few cases (mostly tests). so I disabled the use of dlange and instead use a for loop.